pandas/core/frame.py

```diff
@@ -8455,6 +8455,126 @@ def isin(self, values):
                 self.columns,
             )
 
+    def value_counts(
+        self, normalize=False, sort=True, ascending=False, bins=None, dropna=True
+    ):
+        """
+        Return a Series containing counts of unique rows in the DataFrame.
+
+        .. versionadded:: 1.0.0
+
+        The returned Series will have a MultiIndex with one level per input
```

Comment:
we need to have a ``subset=`` argument (as the 1st arg) to define the columns to group on, default would be all; this is in-line with many other DataFrame methods